### PR TITLE
feature/lpfg 540 content admin

### PR DESCRIPTION
**Pull request by m24murray**
_Basic templating and ci:_
- adding basic templating structure
- adding ci
- adding pagination class and pagination service

_Moving to the new gov front-end kit:_
- migrating to new gov front end (see links below)
- getting rid of the old img/font/js/style dirs
- using core styles (from node_modules/govuk-frontend/assets) and overriding where necessary
- see the following links:
> https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md
> https://design-system.service.gov.uk/styles/page-template/#default

There's a couple of hardcoded values which will be replaced with results from Learning Catalogue API call

### CODE REVIEW

File: src/controllers/home.ts [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 	public index() {
 		return (request: Request, response: Response) => {
-			response.render('index')
+			let searchResults: any = {
+				combinedResults: [],
+				page: 0,
+				results: [1, 2, 3, 4, 5, 6, 7, 8, 9, 0],
+				size: 10,
+				totalResults: 10,
+			}
```
Comment (m24murray) [inline src/controllers/home.ts]: this is hardcoded as placeholder until we integrate the API call to learning catalogue

File: views/page/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <div>
+            <span>Course {{ x + 1 }}</span>
+        </div>
+        {% endfor %}
+        {% if paginatedResults.totalResults !== 0 %}
+        <div class="pager">
+            <div class="pager__controls">
+                {% if paginatedResults.currentPage > 0 %}
+                <a class="pager__prev" href="/search?q={{query}}{courseType ? '&courseType=' + courseType.split(',').join('&courseType=') : ''}&s={{searchResults.size}}&p={{searchResults.page - 1}}">Prev</a>
+                {% endif %}
```
Comment (m24murray) [inline views/page/index.html]: these href's will be changed when we actually integrate the learning catalogue api call

File: views/page/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                            </h1>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% for x in range(10) %}
+        <div>
+            <span>Course {{ x + 1 }}</span>
+        </div>
```
Comment (m24murray) [inline views/page/index.html]: Hardcoding as placeholder until learning catalogue integration

File: views/component/Page.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <base href="/">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>{% block title %}{% endblock %} - Civil Service Learning</title>
+    <link rel="stylesheet" href="/styles/main.css?v=">
```
Comment (m24murray) [inline views/component/Page.html]: I've removed a lot of the previous imports from Page.html (e.g. Google Analytics), but will replace in future as and when we need

File: views/component/Page.html [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+  }) }}
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+    homepageUrl: "#",
+    containerClasses: "govuk-width-container",
+    serviceName: "Civil Service Learning",
+    serviceUrl: "#",
+    navigation: [
```
Comment (elxsy) [inline views/component/Page.html]: INFO - This is the header for UI not management

**Pull request merged**